docs/qmd-knowledge-management.md

```
@@ -202,24 +206,29 @@ qmd embed
 ~/.config/opencode/skill/qmd-knowledge/scripts/record.sh learning "qmd MCP autonomous tool use"
 ```
 
 ### 2. Query Knowledge Later
 
 **You (in Claude):**
+
 > "What have I learned about MCP servers in this project?"
 
 **Claude uses qmd MCP server:**
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)